doc/api/fs.md

```diff
@@ -3843,6 +3846,7 @@ changes:
 
 * `filename` {string|Buffer|URL}
 * `options` {Object}
+  * `bigint` {boolean} **Default:** `false`
```

Comment:
I think we could be clearer in stating what this option is actually doing. Either some more text here like we do for the `fstat()` functions, and/or something in the prose below which discusses the `options` argument.